var/spack/repos/builtin/packages/py-nptyping/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNptyping(PythonPackage):
+    """Type hints for numpy"""
+    homepage = "https://github.com/ramonhagenaars/nptyping"
+    pypi = "nptyping/nptyping-2.4.1.tar.gz"
```

Comment:
The PyPI tarball does not seem to include the `requirements.txt` file, does this package even build?

Reply:
There's even [an issue](https://github.com/ramonhagenaars/nptyping/issues/98) about it - I've linked to the github tarball for now (and installed it, so that one does work). Will revisit once the issue is fixed.

Reply:
Can we remove `pypi` and set `url` globally for now? Keep the comment though, that's useful to refer to if it ever gets fixed.